packages/@aws-cdk/aws-ecs/test/base/test.base-service.ts

```diff
@@ -0,0 +1,24 @@
+import * as cdk from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+import { Cluster } from '../../lib';
+import { ImportedBaseService } from '../../lib/base/base-service';
+
+export = {
+    BaseService: {
+      'can be imported'(test: Test) {
```

Comment:
This test doesn't add much value in my opinion. Tests to witte:

1. Supplying none of `serviceArn` and `serviceName` when importing results in an error in `fromXyzServiceAttributes`.
2. Supplying both `serviceArn` and `serviceName` results in an error in `fromXyzServiceAttributes`.
3. Supplying just the `serviceArn` parses out the correct service name from the ARN.
4. Supplying just the `serviceName` formats the correct service ARN.